js/bitmex.js

```diff
@@ -1022,6 +1023,7 @@ module.exports = class bitmex extends Exchange {
         //   }
         //
         const id = this.safeString (transaction, 'transactID');
+        currency = this.safeCurrencyCode (this.safeString (transaction, 'currency'));
```

Comment:
This looks a bit wrong, i think it should not overwrite the `currency` structure with a currency code here.

Reply:
yeah we can use safeCurrency instead

Reply:
@kroitor in this case, the currency is available in the response, we can't ignore that as well

Reply:
I've added fixes for it.

Reply:
@kroitor nice thanks